monitoring-custom-metrics/deployment/policy.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: security.istio.io/v1beta1
```

Comment:
Why the resource was not needed before? Why it is needed now?

Reply:
The PeerAuthentication is needed to enabled Prometheus to scrape using mTLS. And if you look into the docs of PeerAuthentication from [Istio](https://istio.io/latest/docs/reference/config/security/peer_authentication/), it is the way how it should be defined.